docs/outputs.rst

```diff
@@ -94,6 +94,68 @@ betas_hik_OC_T1c.nii      T1-GS corrected high-kappa components
 meica_mix_T1c.1D          T1-GS corrected mixing matrix
 ======================    =====================================================
 
+Component tables
+----------------
+TEDPCA and TEDICA use tab-delimited tables to track relevant metrics, component
+classifications, and rationales behind classifications.
+TEDPCA rationale codes start with a "P", while TEDICA codes start with an "I".
+
+===============    =============================================================
+Classification     Description
+===============    =============================================================
+accepted           BOLD-like components retained in denoised and high-Kappa data
+rejected           Non-BOLD components removed from denoised and high-Kappa data
+retained           Low-variance components retained in denoised, but not
+                   high-Kappa, data
+===============    =============================================================
+
+TEDPCA codes
+````````````
+
+=====  ===============  ===============================  =======================
+Code   Classification   Description                      Algorithm(s)
+=====  ===============  ===============================  =======================
+P001   rejected         Low Rho, Kappa, and variance     Kundu decision tree
+                        explained
+P002   rejected         Low variance explained           Kundu decision tree
+P003   rejected         Kappa equals fmax                Kundu decision tree
+P004   rejected         Rho equals fmax                  Kundu decision tree
+P101   rejected         Cumulative variance explained    Kundu decision tree
```

Comment:
If we're getting rid of the algorithm column, maybe we could just note in the description something like `(only in stabilized PCA decision tree)`

Reply:
Changing now.